pages/resources/resources.html

```diff
@@ -73,22 +73,22 @@ <h3 class="u-trailer ebook-title">7 Mistakes to Avoid When Hiring Developers</h3
       <div class="card-horizontal overflow-hidden">
         <div class="row">
           <div class="small-12 medium-6 flex-container flex-dir-column align-center padding-3">
-            <h2 class="u-trailer">Browse our Case Studies</h2>
+            <h2 class="u-trailer">Browse our Brand Resources</h2>
 
             <div class="cell">
               <p class="card-text">
-                Learn more about how Esteemed helps companies work better with top digital talent. Browse our collection of video reviews, testimonials, and client stories from our great customers.
+                Learn more about how Esteemed helps companies work better with top digital talent. 
               </p>
               <div class="cell">
-                <a href="/case-studies" class="button" target="_blank">View Case Studies</a>
+                <a href="/for-brands" class="button" target="_blank">Read More</a>
```

Comment:
@Melloboy 
Please be sure to add trailing slashes to all internal links-

/for-brands/